frontend/src/EventDetails/EventDetails.tsx

```diff
@@ -0,0 +1,100 @@
+import classes from "./EventDetails.module.css";
+import Button from "../Button/Button";
+import {
+  ArrowTopRightOnSquareIcon,
+  CalendarIcon,
+  CheckIcon,
+  ClockIcon,
+  MapPinIcon,
+  UserIcon,
+} from "@heroicons/react/24/outline";
+import Keyword from "../Keyword/Keyword";
+import ShareButton from "../ShareButton/ShareButton";
+import { ButtonIcons, ButtonVariants } from "../Button/ButtonTypes";
+
+type EventDetailsProps = {
+  image: string;
+  backgroundPositionY: string;
+  name: string;
+  society: string;
+  date: string;
+  startTime: string;
+  endTime: string;
+  location: string;
+  locationUrl?: string;
+  attending: number;
+  description: string;
+  keywords?: string[];
+};
+
+function EventDetails(props: EventDetailsProps) {
+  return (
+    <div className={classes.container}>
```

Comment:
⚠️ action required: too many `div`s! consider using some of the following elements instead of `div` where applicable throughout the file (in no particular order, you don't have to use all of them):

* `header`
* `main`
* `footer`
* `section`
* `article`

these all function similarly to `<div>` but offer semantic benefits and make your code easier to read.